ci/ray_ci/tester_container.py

```diff
@@ -12,20 +12,29 @@ class TesterContainer(Container):
     A wrapper for running tests in ray ci docker container
     """
 
-    def __init__(self, docker_tag: str) -> None:
+    def __init__(
+        self,
+        docker_tag: str,
+        shard_count: int = 1,
+        shard_ids: Optional[List[int]] = None,
+    ) -> None:
```

Comment:
maybe add some comments on what the args are or something that can explain how to use this class.